Makefile

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+install:
+	go install
+	touch ~/.aliases
```

Comment:
Create a new folder and file `_tpl/.aliases` with the content
```
# ali
## description: Alias manager
## usage: ali ...
## group: core
ali() {
    {{PATH}}/ali $@	
    source ~/.aliases	
    if [[ "$1" == "rm" ]] then	
        unalias $2	
    fi	
}	
```

Replace the `touch` command by 
```
cp -n _tpl/.aliases ~/.aliases
```
this will create the aliases file with the content we want by default.

Once we have the aliases file in place, we need to replace the `{{PATH}}` string by the path to our binary, in this case the current path, which can be extracted from the command `pwd`. A new line calling sed in the makefile like this should do the trick:
```
...
cp -n _tpl/.aliases ~/.aliases
sed -i 's/{{PATH}}/`pwd`/g' ~/.aliases
... 
```

None of this stuff has been tested, so run all the tests you consider useful

Reply:
I'll do my best.